Code/CSharpOOP3/Helper.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace CSharpOOP3
+{
+    public class Helper
+    {
```

Comment:
hmm... I will set a mark to this task, but please review initial text. Helper should be connected to object you created - Flowers

Reply:
done